### DISCUSSION TITLE
We need to support SkyWalking 9.0

### DISCUSSION THREAD

**Discussion by heyanlong** [SkyAPM/SkyAPM-php-sdk#462] [Ideas]
As far as I know, `SkyWalking` has recently been updated to 9.0, We need to follow.

1. skywalking-PHP updated to 9.0
2. skywalking-data-collect-protocol update to 9.0

**Comment by heyanlong** [comment]
At the same time, @AlpherJang will also participate in the development.

**Comment by AlpherJang** [comment]
We can both support old and latest version as controlled with config
